test/data/data-shapes/core/property/property-001.ttl

```diff
@@ -86,15 +86,6 @@ ex:ValidPerson1
           sh:sourceShape ex:PersonShape-address-city ;
           sh:value ex:InvalidCity ;
         ] ;
-      sh:result [
-          rdf:type sh:ValidationResult ;
-          sh:focusNode ex:InvalidAddress ;
-          sh:resultPath ex:city ;
-          sh:resultSeverity sh:Violation ;
-          sh:sourceConstraintComponent sh:ClassConstraintComponent ;
-          sh:sourceShape ex:PersonShape-address-city ;
-          sh:value ex:InvalidCity ;
-        ] ;

```

Comment:
You remove here but these are the official W3C test suites. Not something we want to modify lightly.

On the other hand, both blank nodes have the exact same properties so we'd need to consider this

Reply:
Yes, totally comprehenisble.